app/front/scripts/services/download-package.js

```diff
@@ -80,8 +80,8 @@
               StorageService.clearApplicationState()
                   .then(function() {
                     var packageName = PackageService.getAttributes().name;
-                    var owner = LoginService.email;
-                    state.packagePublicUrl = '/viewer/'+owner+':'+packageName; //dataPackage.uploadUrl;
+                    var owner = LoginService.userid;
```

Comment:
Okay, so I changed it so that userid is the hash of the email and not the provider id.
Either way it's done in the conductor, so there's no change to code in the packager.